types/qrcode/index.d.ts

```diff
@@ -198,7 +198,7 @@ export function toCanvas(
 export function toCanvas(
     canvasElement: HTMLCanvasElement,
     text: string | QRCodeSegment[],
-    options: QRCodeRenderersOptions,
+    options?: QRCodeRenderersOptions,
```

Comment:
If `options` is not provided, it should be the same as the overloaded function at line 182, so it seems not necessary.

Reply:
You're right, but that doesn't cover the case where options is `undefined`.

The [promise definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/373f6a7d14da941996eb3c91dc14084b2f7918a8/types/qrcode/index.d.ts#L193) correctly covers this condition - I'm assuming the callback implementation is nearly identical.

Reply:
`options: QRCodeRenderersOptions | undefined` seems better.